tests/languages/nodejs/Makefile

```diff
@@ -0,0 +1,34 @@
+include ../../common.mk
+
+PROG=/app/index.js
+ENCLAVE_CMD=usr/local/bin/node ${PROG}
+
+DISK_IMAGE=sgxlkl-nodejs.img
+IMAGE_SIZE=1.2G
+
+ifeq ($(SGXLKL_VERBOSE),)
+SGXLKL_ENV=\
+   SGXLKL_MMAP_FILES=Shared SGXLKL_VERBOSE=1 SGXLKL_KERNEL_VERBOSE=0 \
```

Comment:
Is `SGXLKL_MMAP_FILES=Shared` required to run nodejs? If so, why is it not set in the else branch?

Reply:
Added to else block as well.